MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -1769,8 +1737,8 @@ private void showPerson(PersonStatus status, boolean isPrisoner) {
         gridBagConstraints.fill = GridBagConstraints.NONE;
         dialog.getContentPane().add(btn, gridBagConstraints);
 
-        dialog.setMinimumSize(new Dimension(600, 300));
-        dialog.setPreferredSize(new Dimension(600, 300));
+        dialog.setMinimumSize(new Dimension(450, 700));
```

Comment:
Can we replace these fixed values with `dialog.setMinimumSize(UIUtil.scaleForGUI(450, 700)` and `dialog.setPreferredSize(UIUtil.scaleForGUI(450, 700)`. That will ensure the dialog correctly scales with the users scaling settings.

Reply:
Done in all 3 places